wps_tools/testing.py

```diff
@@ -30,28 +30,40 @@ def get(self, **kwargs):
         return super(WpsTestClient, self).get(query)
 
 
-def local_path(file_name):
-    """Return absolute path of file in tests/data directory
+def local_path(sub_filepath):
+    """Return absolute path of file located under tests/data directory
 
     Parameters:
-        file_name (str): File name
+        sub_filepath (str): Sub filepath
 
     Returns:
         str: Absolute local file path
     """
-    return f"file:///{resource_filename('tests', 'data/' + file_name)}"
+    return f"file:///{resource_filename('tests', 'data/' + sub_filepath)}"
 
 
-def opendap_path(file_name):
-    """Return OpenDAP url for file
+def opendap_path(sub_filepath):
+    """Return OpenDAP url for a file located under /storage/data
 
     Parameters:
-        file_name (str): File name
+        sub_filepath (str): Sub filepath
 
     Returns:
         str: OpenDAP url
     """
-    return f"https://docker-dev03.pcic.uvic.ca/twitcher/ows/proxy/thredds/dodsC/datasets/storage/data/projects/comp_support/daccs/{file_name}"
+    return f"https://docker-dev03.pcic.uvic.ca/twitcher/ows/proxy/thredds/dodsC/datasets/storage/data/{sub_filepath}"
+
+
+def http_server_path(sub_filepath):
+    """Return HTTPServer url for a file located under /storage/data
+
+    Parameters:
+        sub_filepath (str): Sub filepath
+
+    Returns:
+        str: HTTPServer url
+    """
+    return f"https://docker-dev03.pcic.uvic.ca/twitcher/ows/proxy/thredds/fileServer/datasets/storage/data/{sub_filepath}"
```

Comment:
Essentially this is the same as `opendap_path`. Perhaps a better way to handle this would be a single method with flags for either http or dods.